src/responder/common/cache_req/plugins/cache_req_common.c

```diff
@@ -140,6 +140,8 @@ cache_req_common_process_dp_reply(struct cache_req *cr,
     }
 
     if (err_maj) {
+        /* does a backtrace make sense here or would the error message
+         * sufficient? */
```

Comment:
Looks like you commited a question :-) IMHO backtrace does not make sense here since the error comes from data provider.

Reply:
Hi,

yes, however I would like to keep the error message in the default log level. @alexey-tikhonov, is it already possible to suppress the backtrace if the debug level is `SSSDBG_OP_FAILURE`?

bye,
Sumit

Reply:
There is `SSSDBG_IMPORTANT_INFO` log level available.

It will print a message if any of bits >=  OP_FAILURE are on (i.e. on default log level as well) but will not trigger a backtrace.

Reply:
Hi,

thanks, I've updated the patches accordingly.

bye,
Sumit